### PR TITLE
`/cfindblock` run clook after clicking coordinates.

### LINKED ISSUE #275: Can I have clook when clicking on coordinates in cfindblock?
I'd like an way to have clook instead of, or in addition to, cglow when clicking on a block coordinate in cfindblock, like how some old versions used to do :-(

### PR DESCRIPTION
**Pull request by L0um15**
Closes #275 
Also display [Glow] component next to coordinates.
image

### CODE REVIEW

File: src/main/resources/assets/clientcommands/lang/en_us.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
 
   "commands.cfindblock.notFound": "No such block found",
   "commands.cfindblock.success.left": "Closest match is at ",
+  "commands.cfindblock.success.glow": " [Glow]",
```
Comment (Earthcomputer) [inline src/main/resources/assets/clientcommands/lang/en_us.json]: Maybe remove the space from here and add it to the code?

Comment (L0um15) [reply src/main/resources/assets/clientcommands/lang/en_us.json]: Sure

**Comment by L0um15** [comment]
Now separator between coordinates and glow component is visible
image

**Comment by Earthcomputer** [comment]
That's fine, thanks!

**Review by Earthcomputer — APPROVED** [review]

**Comment by sudofox** [comment]
For some reason it doesn't run the /clook as a client command when I click the coordinates, only when I type it in manually.

**Pull request merged**